package.json

```diff
@@ -4,34 +4,37 @@
   "files": [
```

Comment:
Will you be publishing on npm after this PR goes though? I'm using it in the dashboard through github, but a regular version might be nice. I assume if not previously published, it doesn't _super_ matter, just a thought.

Reply:
Oh yeh it hasn't been released to npmjs yet. Cool, so I'll likely just release the version as 0.0.1 and then we can go from there. Sound good?

Reply:
Yep, sounds good!